### DISCUSSION TITLE
possible to have a link so as to open a local file?

### DISCUSSION THREAD

**Discussion by efpom** [ransome1/sleek#525] [Ideas]
something along the lines of:
file:///C:/Users/xxx/Desktop/name of file to be opened:

I have done a search to see if the 'problem' has been solved.

**Comment by efpom** [comment]
What I am getting at here, not being a programmer, is that when I have a particular task in my todo.txt file, it would be useful to have a link to a document on my local hard drive that I can access to read it. I know that if I put an email address into the task from sleek, it will open up my email program - thunderbird in my case, with that email as the recipients email address. I also know that if i put a http link into my task that will open up that web address. If that ability could be extended to e.g. Word documents or pdfs, I would find that very useful. What do others think?

**Comment by efpom** [comment]
I think this has been done - see https://github.com/ransome1/sleek/issues/360

Sleek is a fantastic program!

**Reply by ransome1** [reply]
Hi @efpom thanks for starting a discussion. The #360 is only the feature request, not the actual implementation. The link recognition is done by a library called `marked`.

Since I am completely rewriting sleek at the moment, and since I haven't gotten to the point of implementing `marked` yet, I will take a look at it later along the road. 

I remember though, that I already looked into it briefly. If I am not mistaken marked was not able to support the detection of links to local files. But let's see, what we can do.

**Reply by efpom** [reply]
Hello,

Many  thanks for your email message- your sleek program has been a wonderful help for me & my hat's off to you for the amount of effort you have put in to this wonderful program,

Kindest regards @efpom

**Reply by ransome1** [reply]
I'm very glad to hear that. Feel free to spread the word, I am actively looking for developers, who want to help out: https://github.com/ransome1/sleek/discussions/501

Although "just" an interface application, it is very time consuming. I hope that with switching to a modern framework beneath the hood and a significantly lighter backend, there will be a growing number of supporters.
